pkg/apis/mattermost/v1alpha1/clusterinstallation_utils.go

```diff
@@ -308,6 +308,33 @@ func (mattermost *ClusterInstallation) GenerateDeployment(dbUser, dbPassword str
 		},
 	}
 
+	// Mattermost License
+	volumeLicense := []corev1.Volume{}
+	volumeMountLicense := []corev1.VolumeMount{}
+	if mattermost.Spec.MattermostLicenseSecret != "" {
+
+		envVarGeneral = append(envVarGeneral, corev1.EnvVar{
+			Name:  "MM_SERVICESETTINGS_LICENSEFILELOCATION",
+			Value: "/mattermost-license/license",
+		})
+
+		volumeMountLicense = append(volumeMountLicense, corev1.VolumeMount{
+			MountPath: "/mattermost-license",
+			Name:      "mattermost-license",
+			ReadOnly:  true,
+		})
+
+		volumeLicense = append(volumeLicense, corev1.Volume{
+			Name: "mattermost-license",
```

Comment:
Just checking: should this be called `license`?

I see our full path in the env var is `/mattermost-license/license`.

Reply:
this is just the name to match with the volume. but yes the path will be `/mattermost-license` to make it clear and the file is called `license`